angular/test/test-app/e2e/src/overlay-nested.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('Overlay: Nested', () => {
+  beforeEach(() => {
+    cy.visit('/overlay-nested');
+  });
+
+  it('should dismiss the top overlay', () => {
```

Comment:
Is there a reason why this test has to be an Angular-specific test? I feel like we could write this in the core E2E test given that it's an @ionic/core API. There are some existing overlay E2E tests if you are interested: https://github.com/ionic-team/ionic-framework/tree/223f36f6adacf8adce47cee4809a60c94a9e0efa/core/src/utils/test/overlays

Reply:
Swapped it out with the Core test, good suggestion 👍